src/libraries/System.Data.Common/tests/TrimmingTests/DbConnectionStringBuilder.cs

```diff
@@ -134,14 +135,28 @@ public Hashtable GetProperties2()
 
     class TestSite : INestedSite
     {
+        private TestContainer _testContainer = new TestContainer();
+
         public string FullName => null;
         public IComponent Component => throw new NotImplementedException();
-        public IContainer Container => throw new NotImplementedException();
+        public IContainer Container => _testContainer;
```

Comment:
No need to make it more complicated than it has to be.

```suggestion
        public IContainer Container => null;
```